### ISSUE TITLE
Now what? I was the last person to offer any major change to GoFish.

### ISSUE THREAD

**Issue by myearwood1** [mattslay/GoFish#12]
I am willing to support GoFish going forward.

**Comment by DougHennig** [comment]
Thanks for stepping up. The best way to do that it to fork the project, make whatever changes you see fit in the forked repository, then submit a pull request.

**Comment by myearwood1** [comment]
I don't feel any thanks or appreciation from our community anymore. Several
bad actors were not stopped from harassing me.

I submitted a bug fix to Girish years ago. Tamar tried it. The one member
that kept tripping over it could neither debug, fix, nor add my change to
his copy.

I know nothing about git because I have lost ALL motivation to share
anything.

So I have a fork. I can replace an entire method with my change in my
cloned repo and then what?

**Comment by DougHennig** [comment]
Actually, it doesn't look like creating a pull request will help: Matt was the only one who could merge any changes. Send me the files you change (doug@doughennig,com) and I'll merge them into a new location.

**Comment by myearwood1** [comment]
No. I will post my fork and people can get it from my fork directly.

That's what happens when any member of a community is allowed to be
harassed. There is no trust left.
